src/components/InfoPanel.component.js

```diff
@@ -137,6 +171,25 @@ class InfoPanel extends Component {
           </div>
         )}
 
+        {!!media.length &&
+          isEditing &&
+          isRealTimePOI && (
+            <FieldGroup
+              inputType="text"
+              value={captions}
+              placeholder="Change Media Caption Here"
+              onChange={captions =>
+                modifyPOIFormCaption(
+                  this.props.carouselIndex,
+                  captions.target.value
+                )}
+            />
+          )}
+
+        {!!media.length && (
+          <div align="center">{media[carouselIndex].caption}</div>
```

Comment:
you shouldn't be using html to style (referring to the `align="center"`). use css to style